test/unit/self_collectives.c

```diff
@@ -157,23 +197,47 @@ int main(void) {
     if (me == 0) printf(" + all-to-all\n");
 
     in_32 = me; out_32 = -1;
+#ifdef ENABLE_DEPRECATED_TESTS
     shmem_alltoall32(&in_32, &out_32, 1, me, 0, 1, alltoall_psync);
     CHECK("shmem_alltoall32", in_32, out_32);
+#else
+    if (new_team != SHMEM_TEAM_INVALID)
+        shmem_int32_alltoall(new_team, &in_32, &out_32, 1);
+    CHECK("shmem_int32_alltoall", in_32, out_32);
+#endif
     shmem_barrier_all();
 
     in_64 = me; out_64 = -1;
+#ifdef ENABLE_DEPRECATED_TESTS
     shmem_alltoall64(&in_64, &out_64, 1, me, 0, 1, alltoall_psync);
     CHECK("shmem_alltoall64", in_64, out_64);
+#else
+    if (new_team != SHMEM_TEAM_INVALID)
+        shmem_int64_alltoall(new_team, &in_64, &out_64, 1);
+    CHECK("shmem_int64_alltoall", in_64, out_64);
+#endif
     shmem_barrier_all();
 
     in_32 = me; out_32 = -1;
-    shmem_alltoalls32(&in_32, &out_32, 1, 1, 1, me, 0, 1, alltoalls_psync);
+#ifdef ENABLE_DEPRECATED_TESTS
+    shmem_alltoalls32(&in_32, &out_32, 1, 1. 1, me, 0, 1, alltoalls_psync);
     CHECK("shmem_alltoalls32", in_32, out_32);
+#else
+    if (new_team != SHMEM_TEAM_INVALID)
+        shmem_int32_alltoalls(new_team, &in_32, &out_32, 1, 1, 1);
+    CHECK("shmem_int32_alltoalls", in_32, out_32);
+#endif
     shmem_barrier_all();
 
     in_64 = me; out_64 = -1;
-    shmem_alltoalls64(&in_64, &out_64, 1, 1, 1, me, 0, 1, alltoalls_psync);
+#ifdef ENABLE_DEPRECATED_TESTS
+    shmem_alltoalls64(&in_64, &out_64, 1, 1. 1, me, 0, 1, alltoalls_psync);
```

Comment:
Another likely typo:
```suggestion
    shmem_alltoalls64(&in_64, &out_64, 1, 1, 1, me, 0, 1, alltoalls_psync);
```